legal-api/src/legal_api/reports/report.py

```diff
@@ -97,6 +114,30 @@ def _get_report(self):
         if response.status_code != HTTPStatus.OK:
             return jsonify(message=str(response.content)), response.status_code
 
+        create_document = account_id is not None
+        create_filing_types = [
+          'incorporationApplication',
+          'continuationIn',
+          'amalgamation',
+          'registration'
+        ]
+        if self._filing.filing_type in create_filing_types:
+            create_document = create_document and self._business is not None
+            create_document = create_document and self._business.identifier
+            create_document = create_document and self._business.tax_id
```

Comment:
`identifier` will be available when self._business object is not None
```suggestion
            create_document = create_document and self._business and self._business.tax_id
```